### ISSUE TITLE
NTR: Low tetrahydrocortisol to tetrahydrocortisone ratio in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8860]
New term request Low tetrahydrocortisol to tetrahydrocortisone ratio in urine (elthat_130314182321):

**Comment by pnrobinson** [comment]
@MickeySegal
the ratio seems to be usually given the other way around, e.g., https://pubmed.ncbi.nlm.nih.gov/9624595/
What is the medical context for the increased version?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/614662
Biochemically, CORTRD is diagnosed through the assessment of urinary cortisol and cortisone metabolites and consists of measuring the ratio of tetrahydrocortisol (THF) plus 5-alpha-THF to tetrahydrocortisone (THE), which in CORTRD patients is typically less than 0.1 (reference range, 0.7 to 1.2)

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001080
- label: Reduced urine retrahydrocortisol plus 5-alpha-THF to tetrahydrocortisone ratio
- parents: [Abnormal urine hormone level (HP:0012029)](https://hpo.jax.org/app/browse/term/HP:0012029)
- pmid: PMID:21325058
- def: Abnormally low ratio of tetrahydrocortisol (THF) plus 5-alpha-THF to tetrahydrocortisone (THE).
- comment: In cortisone reductase deficiency-2, the ratio of tetrahydrocortisol (THF) plus 5-alpha-THF to tetrahydrocortisone (THE), which in CORTRD patients is typically less than 0.1 (reference range, 0.7 to 1.2).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8860
- synonyms:

**Comment by MickeySegal** [comment]
Typo: retrahydrocortisol

**Comment by pnrobinson** [comment]
fix this

**Comment by pnrobinson** [comment]
fixed typo, thanks @MickeySegal
